fvtest/jitbuildertest/FieldNameTest.cpp

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
```

Comment:
Please update the copyright year :slightly_smiling_face:

Reply:
Fixed